azure/ConsiderationsForServiceDesign.md

```diff
@@ -291,6 +362,9 @@ An action operation that is also long-running combines the [Action Operations](#
 with the [Long Running Operations](#long-running-operations) pattern.
 
 The operation is initiated with a POST operation and the operation path ends in `:<action>`.
+A long-running POST should not be used for resource create -- use PUT as described above.
```

Comment:
```suggestion
A long-running POST should not be used for resource creation: use PUT as described above.
```

Reply:
A colon doesn't seem right to me here, but I'm far from a grammar expert. Can you say why you recommend this over a dash?

Reply:
Put simply, dashes are typically used for a related clause while colons are used to describe or exemplify the preceding text.